.github/workflows/docker.yaml

```
@@ -1,81 +1,62 @@
+        uses: docker/login-action@v3
         with:
-          string: ${{ env.DOCKER_IMAGE }}
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Docker meta
+      - name: Extract metadata for Docker image
         id: meta
-        uses: crazy-max/ghaction-docker-meta@v5
+        uses: docker/metadata-action@v4
```

actionlint (1.7.7)

the runner of "docker/metadata-action@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)